.github/workflows/linting.yml

```diff
@@ -23,12 +23,18 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install --upgrade setuptools
           python -m pip install --upgrade pre-commit
+          python -m pip install --upgrade gitlint
           pre-commit install
       - name: Install cve-bin-tool if needed
         if: ${{ matrix.tool == 'format_checkers' }}
         run: |
           python -m pip install --upgrade wheel
           python -m pip install .
       - name: Run ${{ matrix.tool }}
+        if: ${{ matrix.tool != 'gitlint' }}
         run: |
           pre-commit run ${{ matrix.tool }} --all-files
+      - name: Run gitlint
+        if: ${{ matrix.tool == 'gitlint' }}
+        run: |
+         git log -1 --pretty=%B | gitlint
```

Comment:
I'm not sure if there is a better way to do this. It produces run with steps like the following
![Screenshot from 2022-02-09 16-43-26](https://user-images.githubusercontent.com/63963181/153187600-08a11076-c055-4a79-9bdd-ce36c4c71849.png)
What do you think @Molkree?